tests/command/commands/team_test.py

```diff
@@ -44,8 +68,12 @@ def test_handle_delete():
 def test_handle_create():
     """Test team command create parser."""
     testcommand = TeamCommand()
-    assert testcommand.handle("team create b-s") == "id b-s"
-    assert testcommand.handle("team create b-s --name B") == "team B, id b-s"
+    inputstring = "team create b-s --name 'B S'"
+    outputstring = "new team: b-s, name: B S, "
+    assert testcommand.handle(inputstring) == outputstring
+    inputstring += " --platform web"
+    outputstring += "platform: web, "
```

Comment:
Is a completed line of outputstring intended to end with a `,`?

Reply:
That's how the method is currently written. I'm okay with this for now, because we're only testing parsing.